addons/point_of_sale/models/res_config_settings.py

```diff
@@ -104,6 +104,7 @@ def _default_pos_config(self):
     pos_auto_validate_terminal_payment = fields.Boolean(related='pos_config_id.auto_validate_terminal_payment', readonly=False, string="Automatically validates orders paid with a payment terminal.")
     pos_trusted_config_ids = fields.Many2many(related='pos_config_id.trusted_config_ids', readonly=False)
     point_of_sale_ticket_unique_code = fields.Boolean(related='company_id.point_of_sale_ticket_unique_code', readonly=False)
+    pos_is_closing_entry_with_product = fields.Boolean(related='pos_config_id.is_closing_entry_by_product', readonly=False)
```

Comment:
```suggestion
    pos_is_closing_entry_by_product = fields.Boolean(related='pos_config_id.is_closing_entry_by_product', readonly=False)
```
You forget to rename this.

Reply:
Done.

Reply:
Yes, from my side all changes is done.